### PR TITLE
Added help menu and removed the toast message.

### LINKED ISSUE #5: add a help/info menu
When a user presses f1 (or a button in the UI), we should display a help menu that shows keyboard shortcuts.

```
ctrl+c  copy
ctrl+v  paste
left click to draw pixel
right click to select pixel that is on the canvas
```

We will add more later.

### PR DESCRIPTION
**Pull request by Satharus**
Fixes #5 and #6.

### CODE REVIEW

File: src/input.c [hunk 1]
```diff
@@ -1,9 +1,14 @@
 #include "globals.h"
 #include "file.h"
 
-static void active_color(const unsigned char rect_index)
+// static void active_color(const unsigned char rect_index)
```
Comment (dfirebaugh) [inline src/input.c:4]: Can you go ahead an remove this instead of just commenting it out?

File: src/message_queue.c [hunk 2]
```diff
@@ -94,14 +100,24 @@ void Message_box_render(Message_Queue_t queue)
...
+            message_rect.w = 400;
+            message_rect.h = MESSAGE_BOX_HEIGHT*HELP_OPTIONS;
+            message_rect.x = (SCREEN_WIDTH/2)-(message_rect.w/2);
+            message_rect.y = (SCREEN_HEIGHT/2)-(message_rect.h/2);
+        }
         TTF_Font *Sans = TTF_OpenFont("fonts/OpenSans/OpenSans-SemiBold.ttf", 18);
         SDL_Color White = {255, 255, 255};
-        SDL_Surface *surfaceMessage = TTF_RenderText_Solid(Sans, Message_Queue_front(queue), White);
+        SDL_Surface *surfaceMessage = TTF_RenderText_Blended_Wrapped(Sans, Message_Queue_front(queue), White, message_rect.w);
+//         SDL_Surface *surfaceMessage = TTF_RenderText_Solid(Sans, Message_Queue_front(queue), White);
```
Comment (dfirebaugh) [inline src/message_queue.c:120]: Can you go ahead an remove this instead of just commenting it out?

**Review by dfirebaugh — CHANGES_REQUESTED** [review]

**Comment by dfirebaugh** [comment]
Thanks for the PR!

There seems to be an issue with this.  If press ctrl+c (or hold it down) and then immediately press f1, I end up with something that looks like this.
image

**Comment by Satharus** [comment]
Hey!

I'll get to work on these changes rn. 

The message thing could be fixed by making 2 different message queues. One for help and one for other toasts. I'll check and see how that goes.

**Comment by dfirebaugh** [comment]
Thanks for your work!

**Comment by Satharus** [comment]
No need, buddy!

Check it now :D seems to work fine.
 
Demo

File: src/message_queue.c (hunk 2 above)
Comment (dfirebaugh) [inline src/message_queue.c:120]:
```suggestion
```

File: makefile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 CFLAGS   = -g `sdl2-config --cflags`
-LDFLAGS  = `sdl2-config --libs` -lSDL2 -lSDL2_ttf -lm
+LDFLAGS  = `sdl2-config --libs` -lSDL2 -lSDL2_ttf -lm -z muldefs
```
Comment (dfirebaugh) [inline makefile:2]:
```suggestion
LDFLAGS  = `sdl2-config --libs` -lSDL2 -lSDL2_ttf -lm
```

**Pull request merged**